aws-blog-event-driven-batch-analytics/README.md

```diff
@@ -0,0 +1,164 @@
+# Building Event Driven Batch Analytics on AWS
+
+## Overview
+
+This repository contains the code that supports the [AWS Big Data Blog Post](https://blogs.aws.amazon.com/bigdata/)
+
+### Usecase Description
+Yummy Foods, a hypothetical entity, has franchise stores all over the country. These franchise stores run on heterogeneous platforms and they submit cumulative transaction files to Yummy Foods corporate at various cadence levels through out the day in tab delimited .tdf format. Due to a limitation in some of their systems, some franchise stores occasionally  send additional data starting with characters such as “----“.
```

Comment:
Sorry - maybe we should state 'hypothetical customer'. Also, 'throughout' is one word. You have two spaces between 'ocasionally' and 'send'